test/mockResponses/adequacies.ts

```diff
@@ -0,0 +1,561 @@
+  closest_provider_by_distance: 72,
+  closestProviderByTime: 72
+},
+{
+  id: 180,
+  distance_to_closest_provider: 37.19856946774534,
+  time_to_closest_provider: 55,
+  closest_provider_by_distance: 139,
+  closestProviderByTime: 139
+},
+{
+  id: 190,
+  distance_to_closest_provider: 30.385465456662118,
+  time_to_closest_provider: 27,
+  closest_provider_by_distance: 94,
+  closestProviderByTime: 94
+},
+{
+  id: 200,
+  distance_to_closest_provider: 38.01249479925952,
+  time_to_closest_provider: 44,
+  closest_provider_by_distance: 91,
+  closestProviderByTime: 91
+},
+{
+  id: 210,
+  distance_to_closest_provider: 36.65546104026778,
+  time_to_closest_provider: 22,
+  closest_provider_by_distance: 248,
+  closestProviderByTime: 248
+},
+{
+  id: 220,
+  distance_to_closest_provider: 16.498040497358872,
+  time_to_closest_provider: 50,
+  closest_provider_by_distance: 292,
+  closestProviderByTime: 292
+},
+{
+  id: 230,
+  distance_to_closest_provider: 30.08909094012725,
+  time_to_closest_provider: 27,
+  closest_provider_by_distance: 66,
+  closestProviderByTime: 66
+},
+{
+  id: 240,
+  distance_to_closest_provider: 19.03413116777324,
+  time_to_closest_provider: 46,
+  closest_provider_by_distance: 166,
+  closestProviderByTime: 166
+},
+{
+  id: 250,
+  distance_to_closest_provider: 33.95341377837519,
+  time_to_closest_provider: 39,
+  closest_provider_by_distance: 93,
+  closestProviderByTime: 93
+},
+{
+  id: 260,
+  distance_to_closest_provider: 27.975420387623647,
+  time_to_closest_provider: 42,
+  closest_provider_by_distance: 62,
+  closestProviderByTime: 62
+},
+{
+  id: 270,
+  distance_to_closest_provider: 11.809368803377406,
+  time_to_closest_provider: 42,
+  closest_provider_by_distance: 211,
+  closestProviderByTime: 211
+},
+{
+  id: 280,
+  distance_to_closest_provider: 12.153715375383435,
+  time_to_closest_provider: 53,
+  closest_provider_by_distance: 164,
+  closestProviderByTime: 164
+},
+{
+  id: 290,
+  distance_to_closest_provider: 22.34421349745743,
+  time_to_closest_provider: 50,
+  closest_provider_by_distance: 51,
+  closestProviderByTime: 51
+},
+{
+  id: 300,
+  distance_to_closest_provider: 34.42716848517918,
+  time_to_closest_provider: 36,
+  closest_provider_by_distance: 38,
+  closestProviderByTime: 38
+},
+{
+  id: 310,
+  distance_to_closest_provider: 12.970491488589868,
+  time_to_closest_provider: 41,
+  closest_provider_by_distance: 158,
+  closestProviderByTime: 158
+},
+{
+  id: 320,
+  distance_to_closest_provider: 9.996382386449831,
+  time_to_closest_provider: 24,
+  closest_provider_by_distance: 171,
+  closestProviderByTime: 171
+},
+{
+  id: 330,
+  distance_to_closest_provider: 22.043593863893314,
+  time_to_closest_provider: 13,
+  closest_provider_by_distance: 158,
+  closestProviderByTime: 158
+},
+{
+  id: 340,
+  distance_to_closest_provider: 32.884029775028566,
+  time_to_closest_provider: 55,
+  closest_provider_by_distance: 73,
+  closestProviderByTime: 73
```

Comment:
why are Camel and _
closest_provider_by_distance vs closestProviderByTime ?

Reply:
Gah good catch!